### DISCUSSION TITLE
Error when deploying on QNAP Container Station

### DISCUSSION THREAD

**Discussion by longhurste** [doitandbedone/ispyagentdvr-docker#512] [Q&A] [answered]
I have been running agentdvr on my QNAP for a few years.  My UPS went out on me so I shut down and replaced it.  The external storage that was hosting my snippets/timelapses got unplugged and didn't connect back up.  I connected it back up and tried restarting the container.  It was giving me a "disk full" error, so I blew away the container and started a new one.  The new one is giving me the error: 
Failed to create CoreCLR, HRESULT: 0x80070008

I've tried every which way to restart/rebuild this container, but this error does not give me any references that help.  Thank you in advance for any assistance you can give.

**Comment by ispysoftware** [comment, marked as answer]
found some similar issues, they link to

https://docs.linuxserver.io/faq#jammy

**Comment by longhurste** [comment]
That looks to have solved the issue.  It looks as though QNAP must have to update their firmware to update their Container Station (Docker) before it will be a permanent fix.  Once a new firmware is released, I'll check it again without the --security-opt flag and see if it works without it.  

Thank you for the assistance!
